### PR TITLE
Feat/styles

### PR DESCRIPTION
**Pull request by jaco3n4**

### CODE REVIEW

File: src/search/bar/bar.ts [hunk 1]
```diff
@@ -65,6 +65,7 @@ export class SearchBar extends DefaultControl {
             } else {
                 this.map.searchResults.hide()
             }
+            console.log("str: "+str)
```
Comment (Sparksx) [inline src/search/bar/bar.ts]: Not needed in production

**Comment by mathieugerard** [comment]
IMG_F6D44051CD43-1
- Impossible to click on the reduced mobility on mobile
- The N in the compass is not the same font on mobile and desktop. Desktop is nicer.

**Comment by mathieugerard** [comment]
IMG_D6AD4492A073-1
When going into directions and then going back, the menu button is grey.

**Comment by mathieugerard** [comment]
IMG_941949FC0046-1
Issue with the display of universes. Venue with multiple universes but only 1 language.

**Pull request merged**